### ISSUE TITLE
NPM install @nangohq/frontend is broken

### ISSUE THREAD

**Issue by rguldener** [NangoHQ/nango#612]
Installing `@nanghoq/frontend` is broken. To reproduce:
```
robin@MacBook-Air ~ % mkdir test
robin@MacBook-Air ~ % npm i @nangohq/frontend
npm ERR! code 127
npm ERR! path /Users/robin/node_modules/@nangohq/frontend
npm ERR! command failed
npm ERR! command sh -c npm run build
npm ERR! > @nangohq/frontend@0.15.0 build
npm ERR! > tsc --sourceMap
npm ERR! sh: tsc: command not found

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/robin/.npm/_logs/2023-05-01T20_25_24_563Z-debug-0.log
```

**Comment by rguldener** [comment]
Looks like this change is causing it: https://github.com/NangoHQ/nango/commit/c02b0fe114d75b1a343db49852209e997364a769

**Comment by jesse-olympus** [comment]
@bastienbeurier This is broken in v0.16.0 as well. I recommend versioning the package separately from the fix commits

**Comment by bastienbeurier** [comment]
@jesse-olympus I can't reproce the error. Just ran `npm i @nangohq/frontend` and is worked. Do you mind sharing more info? (e.g. logs)

**Comment by bastienbeurier** [comment]
Sorry actually the fix I did was not in the release. I've just released `0.16.1` that should work.
